test/setup/index.ts

```diff
@@ -92,10 +92,13 @@ export async function deployMonitor(): Promise<string> {
 
     console.log(`platform chosen is ${testPlatform}, createCluster===${createCluster}`);
 
-    await kubectl.downloadKubectl();
+    // TODO: hack, rewrite this
+    const kubernetesVersion = testPlatform === 'openshift3' ? 'v1.11.10' : undefined;
```

Comment:
agreed, I would want this to come from the module responsible for Openshift/Openshift3